examples/__testfixtures__/wl-components/reward-categories/components/styled.output.tsx

```diff
@@ -0,0 +1,209 @@
+import { LinkButton } from 'components/link-button';
+import { primitive } from 'styles/constants/primitives';
+import { filterProps } from 'utils/filter-props';
+import { screenReaderOnly } from 'utils/style/screen-reader-only';
+
+import { RewardsStyleVariant, WithStyleVariant } from '../types';
+
+import { CategoryDropdownProps, OptionsContainerProps } from './types';
+
+export const CategoryTitle = Header.withConfig<{
+  variant: boolean
+}>(p => ({
+  height: '$12',
+  flexWrap: 'wrap',
+  margin: 0,
+  justifyContent: 'center',
+
+  color: p.variant
+    ? '__legacyToken.text-reversed'
+    : '__legacyToken.text-default',
+
+  fontWeight: p.variant ? Styles.fontWeight.normal : 'inherit',
+
+  fontSize: {
+    lg: 19,
+  },
+}));
+
+export const noBottomBorder = {
+  borderWidth: 0,
+  borderColor: 'black',
+  borderStyle: 'solid',
+  borderBottomLeftRadius: 0,
+  borderBottomRightRadius: 0,
+};
+
+export const CategoryContainer = Box.withConfig<{
+  variant: boolean
+}>(p => ({
+  paddingX: '$4',
+  paddingY: 0,
+  borderTopWidth: 1,
+  borderRightWidth: 1,
+  borderBottomWidth: 1,
+  borderLeftWidth: 1,
+  borderStyle: 'solid',
+
+  borderTopColor: p.variant
+    ? primitive.th.$alwaysRedDarken10
+    : '__legacyToken.border-color-default',
+
+  borderRightColor: p.variant
+    ? primitive.th.$alwaysRedDarken10
+    : '__legacyToken.border-color-default',
+
+  borderBottomColor: p.variant
+    ? primitive.th.$alwaysRedDarken10
+    : '__legacyToken.border-color-default',
+
+  borderLeftColor: p.variant
+    ? primitive.th.$alwaysRedDarken10
+    : '__legacyToken.border-color-default',
+
+  borderTopLeftRadius: 8,
+  borderTopRightRadius: 8,
+  borderBottomRightRadius: 8,
+  borderBottomLeftRadius: 8,
+
+  backgroundColor: p.variant
+    ? primitive.th.$alwaysRedDarken10
+    : '__legacyToken.background-pattern',
+
+  // TODO RN: unsupported CSS
+  // boxShadow: '0px 0px 4px rgba(0, 0, 0, 0.1)',
+}));
+
+export const Fieldset = Box.withConfig({
+  borderWidth: 0,
+  borderColor: 'black',
+  borderStyle: 'solid',
+  borderTopWidth: 1,
+  borderTopColor: '__legacyToken.border-color-default',
+  borderTopStyle: 'solid',
+  margin: 0,
+  padding: 0,
+  width: '100%',
+});
+
+export const HiddenLegend = Box.withConfig({
+  ...screenReaderOnly,
+});
+
+export const CategoryDropdownContainer = CategoryContainer.withConfig({
+  width: '100%',
+  padding: 0,
+  // ${({ optionsVisible, overlay }) => overlay && optionsVisible && noBottomBorder};
+});
+
+export const TitleContainer = Box.withConfig({
+  paddingTop: '$2',
+  paddingRight: '$4',
+  paddingBottom: 0,
+  paddingLeft: '$4',
+  width: '100%',
+});
+
+export const Title = ClickableContainer.withConfig<{
+  showBorder: boolean,
+  variant: boolean
+}>(p => ({
+  alignItems: 'space-between',
+  justifyContent: 'center',
+  borderBottomWidth: 1,
+  borderBottomColor: p.showBorder ? '1px' : '0px',
+  borderBottomStyle: 'solid',
+  borderBottomStyle: 'solid',
```

Comment:
2 border bottom styles... think we have a bug here @kfinley-rbi

Reply:
👍 I'll take a look, at the package